cgmml/common/evaluation/CV/height_prediction_with_ml_segmentation.py

```diff
@@ -0,0 +1,67 @@
+import math
+from typing import Tuple
+
+import numpy as np
+from PIL import Image
+
+from cgmml.common.depthmap_toolkit.constants import MASK_CHILD
+from cgmml.common.depthmap_toolkit.depthmap import Depthmap, is_google_tango_resolution
+from cgmml.common.depthmap_toolkit.depthmap_utils import calculate_boundary
+from cgmml.common.background_segmentation.deeplab.deeplab import label_to_color_image
+from cgmml.common.background_segmentation.deeplab.deeplab_model import DEEPLAB_MODEL, PERSON_SEGMENTATION
+
+HEIGHT_SCALE_FACTOR = 0.05
+
+
+def predict_height(depthmap_file: str, rgb_file: str, calibration_file: str) -> Tuple[float, float]:
+
+    # Check if it is captured by a new device
+    dmap = Depthmap.create_from_zip_absolute(depthmap_file, 0, calibration_file)
+    angle = dmap.get_angle_between_camera_and_floor()
+    if is_google_tango_resolution(dmap.width, dmap.height):
+        raise Exception('Skipping because it is not a new device data')
+
+    # Run segmentation
+    im = Image.open(rgb_file).rotate(-90, expand=True)
+    resized_im, seg_map = DEEPLAB_MODEL.run(im)
+
+    # Check if the child's head is fully visible
+    boundary = calculate_boundary(seg_map)
+    if boundary[0] <= 0:
+        raise Exception('Skipping because the child\'s head is not fully visible')
+
+    # Create dummy upscaled depthmap
```

Comment:
The comment can make clear that this actually changes the existing dmap
```suggestion
    # Upscale depthmap
```